docs/user-guide/cli-installcli.md

```diff
@@ -1,109 +1,116 @@
 
-    `zowe-cli-package-<X.Y.Z>.zip` is downloaded to your computer.
+Note, however, that Zowe CLI was not designed for mainframe installation and unexpected behavior can occur.
 
-4. **(Optional)** Meet the [prerequisites](#prerequisites) for installing Zowe CLI plug-ins.
-5. **(Optional)** Navigate to [Download Zowe](https://www.zowe.org/download.html) and click the **Zowe \<X.Y.Z\> CLI Plugins** button to download the plug-ins.
+:::caution
+   
+Installing Zowe CLI on the mainframe is not supported by Zowe conformant support providers. By choosing this installation method, you need to perform your own independent troubleshooting if any problems arise.  
+   
+:::
 
-6. **(Optional)** Read the End User License Agreement for Zowe plug-ins and click **I agree** to download the plugins package.
+### Installing Zowe CLI on z/Linux
 
-    `zowe-cli-plugins-next-<X.Y.Z>.zip` is downloaded to your computer.
+Installation steps for Zowe CLI depend on whether your z/Linux environment requires the secure credential storage.
 
-7. Unzip the contents of `zowe-cli-package-<X.Y.Z>.zip` (and optionally `zowe-cli-plugins-<X.Y.Z>.zip`) to a working directory.
+#### Installing with secure credential storage
 
-8. To install Zowe CLI Core, open a command-line window and issue the following commands to the working directory that you used in Step 7:
+   1. Follow the steps in [Configuring secure credential storage on headless Linux operating systems](./cli-configure-scs-on-headless-linux-os.md).
+   2. Follow the steps in [Installing Zowe CLI and Zowe CLI plug-ins from an npm online registry](#installing-zowe-cli-and-zowe-cli-plug-ins-from-an-npm-online-registry) or [Installing Zowe CLI and Zowe CLI plug-ins from a local package](#installing-zowe-cli-and-zowe-cli-plug-ins-from-a-local-package).
 
-   ```
-   npm install -g zowe-cli.tgz
-   ```
+#### Installing without secure credential storage
 
-   :::note
-   
-   If an `EACCESS` error displays, see [Resolving EACCESS permissions errors when installing packages globally](https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally) in the npm documentation.
+   1. Follow the steps in [Installing Zowe CLI and Zowe CLI plug-ins from an npm online registry](#installing-zowe-cli-and-zowe-cli-plug-ins-from-an-npm-online-registry) or [Installing Zowe CLI and Zowe CLI plug-ins from a local package](#installing-zowe-cli-and-zowe-cli-plug-ins-from-a-local-package).
+   2. Follow the steps in [Configuring Zowe CLI where secure credential storage is not available](./cli-configure-cli-on-os-where-scs-unavailable.md).
 
-   :::
+### Installing Zowe on a USS system, or O/S without secure credential storage
```

Comment:
I think this should be changed to something like: "Installing Zowe CLI on z/OS USS, or an OS without access to secure credential storage"

Mainly, we should change Zowe to Zowe CLI and remove the slash in "O/S"